flink-runtime/src/main/java/org/apache/flink/runtime/util/ZooKeeperUtils.java

```diff
@@ -244,6 +244,7 @@ public static ZooKeeperLeaderElectionService createLeaderElectionService(
 	 * @return {@link ZooKeeperSubmittedJobGraphStore} instance
 	 * @throws Exception if the submitted job graph store cannot be created
 	 */
+	@Deprecated
```

Comment:
Can we remove this method? If not, then at least write why this method was deprecated using the `@deprecated` tag.